dlp/src/main/java/com/example/dlp/Inspect.java

```diff
@@ -756,28 +597,7 @@ public static void main(String[] args) throws Exception {
     }
 
     // string inspection
-    if (cmd.hasOption("s")) {
-      String val = cmd.getOptionValue(stringOption.getOpt());
-      inspectString(
-          val,
-          minLikelihood,
-          maxFindings,
-          infoTypesList,
-          customInfoTypesList,
-          includeQuote,
-          projectId);
-    } else if (cmd.hasOption("f")) {
-      String filePath = cmd.getOptionValue(fileOption.getOpt());
-      inspectFile(
-          filePath,
-          minLikelihood,
-          maxFindings,
-          infoTypesList,
-          customInfoTypesList,
-          includeQuote,
-          projectId);
-      // gcs file inspection
-    } else if (cmd.hasOption("gcs")) {
+    if (cmd.hasOption("gcs")) {
```

Comment:
I don't know if this is in scope or just because the executable already exists in this project, but are we going to continue to surface executables with new samples? Last I heard it was "no," but just wanted to confirm.

Reply:
No, the new format puts the nix on future CLIs